x-pack/plugins/observability/public/components/shared/exploratory_view/series_builder/columns/chart_types.test.tsx

```diff
@@ -7,14 +7,14 @@
 
 import React from 'react';
 import { fireEvent, screen, waitFor } from '@testing-library/react';
-import { SeriesChartTypes, XYChartTypes } from './chart_types';
 import { mockUrlStorage, render } from '../../rtl_helpers';
+import { SeriesChartTypesSelect, XYChartTypesSelect } from './chart_types';
 
-describe.skip('SeriesChartTypes', function () {
+describe.skip('SeriesChartTypesSelect', function () {
```

Comment:
i had skipped these as well because of getByRole slowness, i wasn't earlier sure why it's happening so make CI happy, i did that, will unskip in a follow up.